### ISSUE TITLE
Homebridge dependency warning

### ISSUE THREAD

**Issue by oznu** [ztalbot2000/homebridge-cmd4#51]
Hi there,

We've noticed your plugin is using `homebridge` as a prod-dependency.

Homebridge should only be included in plugins as a dev-dependency and used for TypeScript types.

In the [next release](https://github.com/homebridge/homebridge/blob/beta/CHANGELOG.md) of we are changing Homebridge to only [export types](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-8.html#-type-only-imports-and-export) and [const enums](https://www.typescriptlang.org/docs/handbook/enums.html#const-enums) which can be safely used in your code and not result in Homebridge becoming a runtime dependency. 

You can test this out using the current beta of Homebridge:

```
npm install --save-dev homebridge@beta
```

**Comment by ztalbot2000** [comment]
I will investigate.  As I remember it, I put it that way so that unit testing of my structure can use Homebridge to query any changes you make.  I will look into this further.

**Comment by jsiegenthaler** [comment]
This is probably why homebridge complains when it starts up:
```

[6/17/2020, 11:03:59] The plugin "homebridge-cmd4" defines 'homebridge' and/or 'hap-nodejs' in their 'dependencies' section, meaning they carry an additional copy of homebridge and hap-nodejs. This not only wastes disk space, but also can cause major incompatibility issues and thus is considered bad practice. Please inform the developer to update their plugin!
[6/17/2020, 11:03:59] Loaded plugin: homebridge-cmd4@2.3.1
[6/17/2020, 11:03:59] Registering accessory 'homebridge-cmd4.Cmd4'
[6/17/2020, 11:03:59] Registering platform 'homebridge-cmd4.Cmd4'


```

**Comment by ztalbot2000** [comment]
This is a new message by homebridge. I probably just skipped over it. this
message also refers to another issue so is a duplicate, but i will not
close this one either because I like the warning and will see if it goes
away when the other issue is fixed.

thanks,
john Talbot

**Comment by ztalbot2000** [comment]
Fixed with commit 810a7ae
